docs/docs/fargate-tutorials/prisma.md

```diff
@@ -134,126 +140,214 @@ node index.js
+
+```javascript
+[
+  {
+    action: 'create',
+    table_name: 'log_group',
+    id: '',
+    description: 'quickstart-log-group'
+  },
+  {
+    action: 'create',
+    table_name: 'repository',
+    id: '',
+    description: 'quickstart-repository'
+  },
+  {
+    action: 'create',
+    table_name: 'role',
+    id: '',
+    description: 'quickstart-ecs-task-exec-role'
+  },
+  {
+    action: 'create',
+    table_name: 'security_group',
+    id: '16',
+    description: '16'
+  },
+  {
+    action: 'create',
+    table_name: 'security_group_rule',
+    id: '15',
+    description: '15'
+  },
+  {
+    action: 'create',
+    table_name: 'security_group_rule',
+    id: '16',
+    description: '16'
+  },
+  {
+    action: 'create',
+    table_name: 'listener',
+    id: '8',
+    description: '8'
+  },
+  {
+    action: 'create',
+    table_name: 'load_balancer',
+    id: '',
+    description: 'quickstart-load-balancer'
+  },
+  {
+    action: 'create',
+    table_name: 'target_group',
+    id: '',
+    description: 'quickstart-target'
+  },
+  {
+    action: 'create',
+    table_name: 'cluster',
+    id: '',
+    description: 'quickstart-cluster'
+  },
+  {
+    action: 'create',
+    table_name: 'task_definition',
+    id: '8',
+    description: '8'
+  },
+  {
+    action: 'create',
+    table_name: 'service',
+    id: '',
+    description: 'quickstart-service'
+  },
+]
 ```
 
-6. Grab your load balancer DNS and access your service!
+Now grab your load balancer DNS and access your service!
 ```bash
 QUICKSTART_LB_DNS=$(psql -At 'postgres://d0va6ywg:nfdDh#EP4CyzveFr@db.iasql.com/_4b2bb09a59a411e4' -c "
 SELECT dns_name
 FROM load_balancer
 WHERE load_balancer_name = '<project-name>-load-balancer';")
-```
 
-7. Connect to your service!
-
-```
 curl ${QUICKSTART_LB_DNS}:8088/health
 ```
 
 ## Delete managed cloud resources
 
 :::warning
 
-If you did not create a new account this section will delete **all** records managed by IaSQL, including the ones that previously existed in the account under any of the used modules. Run `SELECT * FROM iasql_plan_apply()` after `SELECT delete_all_records();` and before `SELECT iasql_apply();` to get a preview of what would get deleted. To undo `SELECT delete_all_records();`, simply run `SELECT iasql_sync();` which will synchronize the database with the cloud's state.
+If you did not create a new AWS account this section will delete **all** records managed by IaSQL, including the ones that previously existed in the account under any of the used modules. Run `SELECT * FROM iasql_plan_apply()` after `SELECT delete_all_records();` and before `SELECT iasql_apply();` to get a preview of what would get deleted. To undo `SELECT delete_all_records();`, simply run `SELECT iasql_sync();` which will synchronize the database with the cloud's state.
```

Comment:
Not for this PR, but with it down to just `codebuild` and `ecs_simplified`. Is it possible to provide a hand-written set of `DELETE` statements here? I am pretty sure `ecs_simplified` will do the right thing if you delete its record and drop all of the related records at the same time in the other modules, right?

Reply:
https://github.com/iasql/iasql-engine/issues/1335 and yes